### PR TITLE
content: Add description for Contact Us section of Footer

### LINKED ISSUE #77: Content: Add a description for Contact Us Section of Footer
## Description
As a developer,
I need a short description for the  contact us section,
so that I can replace the dummy text field.

## Mocks
Footer

### Update [Required]
- [ ] The content should be consistent with the design and be clear enough so that a user can communicate effectively to the contact details provided as and when required.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
2 hours

### PR DESCRIPTION
**Pull request by siddhipandare**
### Content for Contact Us section of Footer

Added content for Contact Us section of the Footer in src/contact_us.json file

Fixes #77

### Type of Change:
- Documentation

### How Has This Been Tested?

Checked if the JSON is valid or not using [jsonlint.com](https://jsonlint.com/)
image

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] Any dependent changes have been merged

### CODE REVIEW

**Comment by siddhipandare** [comment]
> Please move it under src/content folder.

Done.

File: src/content/contact_us.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+            "title": "Contact Us",
+            "content": [
+                {
+                    "par": "Have a question or want to suggest improvements for our project? We can help."
+                },
+                {
+                    "par": "Head to AnitaB.org Open Source Zulip Chat @https://anitab-org.zulipchat.com and get your doubts cleared in #questions stream"
+                },
+                {
+                    "par": "Each active project has its own stream,  #vms,#mentorship-system,#portal and #power-up, for discussions"
```
Comment (sakshi1499) [inline src/content/contact_us.json:13]: Can you not have a comma after power-up and give space after every comma?

File: src/content/contact_us.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+    "sections": [
+        {
+            "title": "Contact Us",
+            "content": [
+                {
+                    "par": "Have a question or want to suggest improvements for our project? We can help."
+                },
+                {
+                    "par": "Head to AnitaB.org Open Source Zulip Chat @https://anitab-org.zulipchat.com and get your doubts cleared in #questions stream"
```
Comment (sakshi1499) [inline src/content/contact_us.json:10]: I think its head over to or you can write go to, wouldn't that be better?

File: src/content/contact_us.json (hunk 1 above)
Comment (siddhipandare) [reply src/content/contact_us.json:13]: yes surely!

File: src/content/contact_us.json (hunk 2 above)
Comment (siddhipandare) [reply src/content/contact_us.json:10]: Yes it would me. Changing it!

File: src/content/contact_us.json (hunk 1 above)
Comment (siddhipandare) [reply src/content/contact_us.json]: Done 👍

File: src/content/contact_us.json (hunk 2 above)
Comment (siddhipandare) [reply src/content/contact_us.json]: Done 👍

File: src/content/contact_us.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+            "title": "Contact Us",
+            "content": [
+                {
+                    "par": "Have a question or want to suggest improvements for our project? We can help."
+                },
+                {
+                    "par": "Go to AnitaB.org Open Source Zulip Chat @https://anitab-org.zulipchat.com and get your doubts cleared in #questions stream"
+                },
+                {
+                    "par": "Each active project has its own stream, #vms, #mentorship-system, #portal and #power-up for discussions"
```
Comment (foongminwong) [inline src/content/contact_us.json:13]: @siddhipandare your paragraphs look good to me! here are some phrases/ideas if you want to add/make changes:
- Having a question in your mind? We are here to help! Post your question under #questions stream in our Zulip Chat @https://anitab-org.zulipchat.com 
- Each active project has its own discussion stream such as #vms, #mentorship-system, #portal, and #powerup. 
- AnitaB.org Open Source is growing globally. Any feedback or improvement suggestions for our projects are most welcome.

Comment (siddhipandare) [reply src/content/contact_us.json:13]: This sounds better! Will do 👍

Comment (siddhipandare) [reply src/content/contact_us.json]: I figured that my tone was not as welcoming. I have changed it a bit. Let me know if it is too much 😓

File: src/content/contact_us.json (hunk 1 above)
Comment (annabauza) [reply src/content/contact_us.json]: Power up has been archived

Comment (annabauza) [reply src/content/contact_us.json]: We have open source programs, bridge in tech and stem diverse tv

Comment (siddhipandare) [reply src/content/contact_us.json]: > We have open source programs, bridge in tech and stem diverse tv

Corrected that 👍

**Pull request merged**